api/src/test/java/org/motechproject/nms/api/osgi/KilkariControllerBundleIT.java

```diff
@@ -272,6 +279,25 @@ public void testCreateSubscriptionRequestDifferentPacks() throws IOException, In
     }
 
     @Test
+    public void testCreateSubscriptionsNoLanguageInDB() throws IOException, InterruptedException {
+        setupData();
```

Comment:
It only makes sense to have a @before if you need the database to be in the exact same state for each and every test, which is not always the case.